configuration.go

```diff
@@ -270,6 +272,12 @@ func NewTraefikDefaultPointersConfiguration() *TraefikConfiguration {
 	defaultDocker.ExposedByDefault = true
 	defaultDocker.Endpoint = "unix:///var/run/docker.sock"
 
+	//default Swarm
+	var defaultSwarm provider.Swarm
+	defaultSwarm.Watch = true
+	defaultSwarm.ExposedByDefault = true
+	defaultSwarm.Endpoint = "unix:///var/run/docker.sock"
```

Comment:
@diegofernandes I understand as I was first person who posted the issue https://github.com/containous/traefik/issues/504 (😄 ). However, it appears that the PR doesn't work yet because it doesn't seem to read the `label` from the service yet. This might be to wrong place to put my comments. This is a general comment for the PR.